nilearn/plotting/displays.py

```diff
@@ -1242,37 +1242,45 @@ def find_cut_coords(self, img=None, threshold=None, cut_coords=None):
                           for c in sorted(self._cut_displayed)]
         return cut_coords
 
+    def _find_inital_axes_coord(self,index):
```

Comment:
There should be a space after `,` when specifying parameters or list elements, or anywhere there's a comma really.
`(self, index)`
Those are the PEP8 rules and they aim to enhance code readability. Google and read up on them, they are pretty simple and sensible.